src/frontend/magnify/packages/components/src/utils/settings.ts

```diff
@@ -1,5 +1,16 @@
-// Keycloak
+declare global {
+  interface Window {
+    config: {
+      API_URL: string;
+      KEYCLOAK_CLIENT_ID: string;
+      KEYCLOAK_URL: string;
+      KEYCLOAK_EXPIRATION_SECONDS: number;
+      KEYCLOAK_REALM: string;
+      JITSI_URL: string;
+    };
+  }
+}
+
+window.config = window.config || {};
```

Comment:
This is defined twice in two packages. Can we factorize ?
Furthermore, I think this logic should be moved into a dedicated module instead within the settings module.

Reply:
agreed